core/src/main/java/org/eclipse/hono/util/RequestResponseApiConstants.java

```diff
@@ -43,20 +41,23 @@
     public static final String FIELD_PAYLOAD   = "payload";
     public static final String FIELD_TENANT_ID = "tenant-id";
 
-    /* request actions */
-    public enum Action {
+    /**
+     * Request standard actions to support the standard lifecycle.
+     * If more or other actions shall be used, an own enum type should be defined.
+    */
+    public enum StandardAction implements RequestResponseAction {
```

Comment:
what is wrong with simply defining the actions that are defined for an API in the corresponding `*Constants` class?

Reply:
Basically, not too much anymore, except for: 
it would be completely identical for tenant and credentials API, and it would double about 30 code lines.
Since it often is probably the case that the HTTP APIs have get,add,update,delete, I have put it into the StandardAction.

Previously, there were generic methods operating on this type. Due to the introduced handlers this does not exist anymore, so the left reason to keep them here is purely the code duplication.